src/esp/scene/SceneNode.h

```diff
@@ -10,7 +10,6 @@
 #include <Corrade/Containers/Containers.h>
 #include <Corrade/Containers/Optional.h>
 #include <Magnum/Math/Range.h>
-#include <Magnum/SceneGraph/Object.h>
```

Comment:
Why? `SceneNode` depends on `SceneGraph::Object`, it's just typedef'd to `MagnumObject`.